### ISSUE TITLE
Unsupported built-in function: GENERATE_UUID

### ISSUE THREAD

**Issue by struckoff** [GoogleCloudPlatform/cloud-spanner-emulator#135]
Hi! It seems like GENERATE_UUID function is not supported by the latest version([v1.5.10](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/releases/tag/v1.5.10)) emulator. Are there any plans on adding the function?

### Run emulator
```
docker run -p 9010:9010 -p 9020:9020 gcr.io/cloud-spanner-emulator/emulator:1.5.10
```

### Run DDL

```
CREATE TABLE items (
  uid STRING(36) DEFAULT (GENERATE_UUID()),
  id STRING(MAX) NOT NULL,
) PRIMARY KEY (uid)
```

Migrate with golang-migrate

```
migrate -path sql/ -database spanner://${DSN} -verbose up
```

### Get error

```
error: migration failed in line 0: CREATE TABLE items (
  uid STRING(36) DEFAULT (GENERATE_UUID()),
  id STRING(MAX) NOT NULL,
) PRIMARY KEY (uid) (details: rpc error: code = InvalidArgument desc = Error parsing the default value of column `items`.`uid`: Unsupported built-in function: GENERATE_UUID)
```

**Comment by olavloite** [comment]
Yes, this function is being worked on for the emulator. We don't have a concrete timeline at the moment. I'll report back here when that is known.

**Comment by olavloite** [comment]
@struckoff The current ETA for this function for the emulator is mid-November 2023.

**Comment by struckoff** [comment]
This is good news, thank you

**Comment by taka-oyama** [comment]
For anyone reading this issue, I confirmed that this was implemented in 1.15.13.
